src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/EventValidator.java

```diff
@@ -13,41 +14,97 @@
 public class EventValidator {
 
     /**
-     * Validates the events timestamps are in the correct order and the updated event isn't already past.
+     * Validates the events timestamps are in the correct order and the updated event can be updated,
+     * i.e is not a mandatory or reminder event.
      * @param existingEvents
      * @param updatedEvent
      * @return
      */
-    public boolean validate(final List<Event> existingEvents, final Event updatedEvent) {
+    public boolean validate(final List<Event> existingEvents, final Event updatedEvent,
+                            final CollectionExerciseDTO.CollectionExerciseState collectionExerciseState) {
+
+        // Can only update reminders of the non mandatory events when READY_FOR_LIVE
+        if (collectionExerciseState.equals(CollectionExerciseDTO.CollectionExerciseState.READY_FOR_LIVE)
+                && (!isReminder(updatedEvent) && !isMandatory(updatedEvent))) {
```

Comment:
I thought a reminder can be set even after the CE is READY_FOR_LIVE/ LIVE.

Reply:
Nope https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/259555593/US076+INT+Amend+Variable+Event+Dates

Reply:
When the collection exercise has been set 'Ready for Live', the user is only able to amend the reminder dates

Given: The internal user has set the collection exercise as 'Ready for Live'
When: They navigate to the collection exercise details page
Then: They are able to edit the reminder dates (only)